service/src/main/resources/application.yml

```diff
@@ -30,7 +30,7 @@ env:
     tps: ${TPS_ADDRESS:https://tps.dsde-dev.broadinstitute.org/}
     lzs: ${LZS_ADDRESS:https://landingzone.dsde-dev.broadinstitute.org/}
   azure:
-    environment: ${AZURE_ENVIRONMENT:AZURE}
+    environment: ${AZURE_ENVIRONMENT:AzureCloud}
```

Comment:
inspired by https://github.com/DataBiosphere/terra-landing-zone-service/pull/536. This value didn't cause more problems because there's a default case in https://github.com/DataBiosphere/terra-workspace-manager/blob/2fb17c499a2fc9cd72c785cdc0ffe2104e91f2dd/service/src/main/java/bio/terra/workspace/app/configuration/external/AzureConfiguration.java#L35-L39. However, this value is passed as-is to the database utils, where I think it _will_ cause a problem; the database utils yaml file expects `AzureCloud`, not `AZURE` in https://github.com/DataBiosphere/terra-workspace-manager/pull/1806/files#diff-48a6c4ffa6ec09eb28b0d58dee997ddf20fa5ff4c596a10e0d83a4f4b68a43bf.